src/main.cc

```diff
@@ -721,8 +721,22 @@ main(int argc, char **argv)
 		script = argv[0];
 		title_set_script_name(argv[0]);
 	}
+	strlcpy(tarantool_path, tarantool_bin, sizeof(tarantool_path));
+	if (strlen(tarantool_path) < strlen(tarantool_bin))
+		panic("executable path is trimmed");
 
-	crash_init(tarantool_bin);
+	/*
+	 * We need to keep clock data locally to report uptime without binding
+	 * to libev and etc. Because we're reporting information at the moment
+	 * when crash happens and we are to be independent as much as we can.
+	 */
+	struct timespec ts;
+	if (clock_gettime(CLOCK_MONOTONIC, &ts) == 0)
+		box_start_time = ts.tv_sec;
+	else
+		say_syserror("fail to get box_start_time, ignore");
+
+	crash_callback = box_crash_callback;
```

Comment:
IMO, better set it in `box_init` - anyway, feedback won't be enabled until the box is configured.

Reply:
Done.